CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to GenLayer Simulator
+
+We're thrilled that you're interested in working on the GenLayer Simulator! This document will guide you through the contribution process.
+
+## What is the GenLayer Simulator?
+
+The GenLayer Simulator is an interactive sandbox designed for developers to explore the potential of GenLayer's Intelligent Contracts. It replicates the GenLayer network's execution environment and consensus algorithm, providing a controlled and local environment to test different ideas and behaviors.
+
+Begin by setting up your development environment to start contributing: Follow our [README](https://github.com/yeagerai/genlayer-simulator/blob/main/README.md) for detailed setup instructions.
```

Comment:
This is not necessary to 3 of the 4 ways of contributing. I would take it out from here and and cleared steps for each individual way of contributing

Reply:
okay